dev/arktype.io/docs/api/arrayof.md

```diff
@@ -4,8 +4,24 @@ hide_table_of_contents: true
 
 # arrayOf
 
-## text
+## operator
 
-```ts
-arrayOf: Ark["arrayOf"]
-```
+-   [arrayOf](./arrayof.md)
+
+## string
+
+-   "type[]"
+
+## tuple
+
+-   ["arrayOf", &lt;object&gt;]
+
+## helper
+
+-   arrayOf(&lt;object&gt;)
+
+## example
+
+-   const arrayOf = type("number[]") <br/>
```

Comment:
Can we include the examples for each variant of the operator in that section of the docs? I think that'd make it much easier to see what was going on with T